master/internal/command/command.go

```diff
@@ -172,6 +172,11 @@ func (c *Command) Start(ctx context.Context) error {
 	if err := c.persist(); err != nil {
 		c.syslog.WithError(err).Warnf("command persist failure")
 	}
+
+	err = task.InsertNTSCAllocationWorkspaceRecord(ctx, c.allocationID)
+	if err != nil {
+		return fmt.Errorf("failure while inserting NTSC allocation workspace record: %w", err)
```

Comment:
Fair point, are we wanting to persist this information only if we know an allocation was successful? On second thought, it might be worth while to go head and store an entry regardless since we will likely be reporting information on errored allocations. 

I would imagine failing to persist this information should cause the allocation to fail?